README.md

```diff
@@ -1,34 +1,148 @@
-# Hello World
-Hello World is a project about Introduction to Microverse
-# Built With
-it is built with Html & CSS
-# Key Features
-Html
-css
-# Getting Started
+<a name="readme-top"></a>
+
+
+
+<div align="center">
+
+  <img src="https://github.com/microverseinc/readme-template/blob/master/murple_logo.png" alt="logo" width="140" height="auto" />
+  <br/>
+
+  <h3><b>README.md</b></h3>
+
+</div>
+
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+ - [🛠 Built With](#built-with)
+      - [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
```

Comment:
- [ ] This README file was provided as a template to guide you in documenting your project professionally. Here at Microverse, we encourage our students to be professional in documenting their projects. So for all your projects going forward, you should always edit this README template to suit your project. Please remove the square brackets and edit your title like below. E.g 👇🏾 

```html
# 📖 <a name="about-project">'Hello Microverse' project</a>
```
#### **_Please note that where ever you see these square brackets anywhere in the README file, you should edit the text and remove the brackets afterward._**